src/components/FormSelectList.vue

```diff
@@ -197,7 +201,11 @@ export default {
       }
 
       // If no endpoint has been specified, do not make the api call
-      if (!!selectedEndPoint && selectedEndPoint.toString().trim().length === 0) {
+      if (
+        selectedEndPoint === null ||
+        typeof selectedEndPoint === "undefined" ||
```

Comment:
```suggestion
        !!selectedEndPoint ||
```

Reply:
truth tables are not the same:

| selectedEndpoint | selectedEndpoint === null \|\| typeof selectedEndpoint === "undefined" | !!selectedEndpoint |
| ------------------ | ------------------------------------------------------------------------ | -------------------- |
| null               | true                                                                     | false                |
| undefined          | true                                                                     | false                |
| object             | false                                                                    | true                 |